docs/v1.X/docs/getting_started/installation.md

```diff
@@ -25,35 +35,54 @@ During intensive logs processing, CPU is going to be the most used resource, and
 However, running [metabase](https://www.metabase.com/) (the dashboard deployed by `cscli dashboard setup`) [requires 1-2Gb of RAM](https://www.metabase.com/docs/latest/troubleshooting-guide/running.html).
 
 
-# Install using crowdsec repository
+# Install on debian using crowdsec repository
+
+On debian and ubuntu, packages are hosted on [packagecloud.io](https://packagecloud.io).
 
 Crowdsec distributes their own pragmatic debian packages that closely follow the development stream (packages are automatically published on release), and are suitable for those that want to keep up with the latest changes of crowdsec.
 
+
 ## setup the repository
 
+Instructions for adding repositories to your machine can be found in [packagecloud's installation docs](https://packagecloud.io/crowdsec/crowdsec/install#bash-deb).
+
+If you're not fond of `curl ... | sudo bash`, follow instruction bellow :
+
+ 1. Retrieve the signing key 
+
 ```bash
-wget -qO - https://s3-eu-west-1.amazonaws.com/crowdsec.debian.pragmatic/crowdsec.asc |sudo apt-key add - && echo "deb https://s3-eu-west-1.amazonaws.com/crowdsec.debian.pragmatic/$(lsb_release -cs) $(lsb_release -cs) main" | sudo tee /etc/apt/sources.list.d/crowdsec.list > /dev/null
-sudo apt-get update
+curl https://packagecloud.io/gpg.key | sudo apt-key add -
```

Comment:
Wrong URL.
Should be `curl -L https://packagecloud.io/crowdsec/crowdsec/gpgkey | sudo apt-key add -`